exercises/practice/two-bucket/tests/two_bucket_test.bal

```diff
@@ -0,0 +1,142 @@
+import ballerina/test;
+
+@test:Config
+function test1() {
+    // Measure using bucket one of size 3 and bucket two of size 5 - start with bucket one
+    TwoBucketResult result = measure(
+        bucketOne = 3,
+        bucketTwo = 5,
+        goal = 1,
+        startBucket = "one"
+    );
+    TwoBucketSolution expected = {moves: 4, goalBucket: "one", otherBucket: 5};
+    test:assertEquals(result, expected);
+}
+
+@test:Config {
+    enable: false
+}
+function test2() {
+    // Measure using bucket one of size 3 and bucket two of size 5 - start with bucket two
+    TwoBucketResult result = measure(
+        bucketOne = 3,
+        bucketTwo = 5,
+        goal = 1,
+        startBucket = "two"
+    );
+    TwoBucketSolution expected = {moves: 8, goalBucket: "two", otherBucket: 3};
+    test:assertEquals(result, expected);
+}
+
+@test:Config {
+    enable: false
+}
+function test3() {
+    // Measure using bucket one of size 7 and bucket two of size 11 - start with bucket one
+    TwoBucketResult result = measure(
+        bucketOne = 7,
+        bucketTwo = 11,
+        goal = 2,
+        startBucket = "one"
+    );
+    TwoBucketSolution expected = {moves: 14, goalBucket: "one", otherBucket: 11};
+    test:assertEquals(result, expected);
+}
+
+@test:Config {
+    enable: false
+}
+function test4() {
+    // Measure using bucket one of size 7 and bucket two of size 11 - start with bucket two
+    TwoBucketResult result = measure(
+        bucketOne = 7,
+        bucketTwo = 11,
+        goal = 2,
+        startBucket = "two"
+    );
+    TwoBucketSolution expected = {moves: 18, goalBucket: "two", otherBucket: 7};
+    test:assertEquals(result, expected);
+}
+
+@test:Config {
+    enable: false
+}
+function test5() {
+    // Measure one step using bucket one of size 1 and bucket two of size 3 - start with bucket two
+    TwoBucketResult result = measure(
+        bucketOne = 1,
+        bucketTwo = 3,
+        goal = 3,
+        startBucket = "two"
+    );
+    TwoBucketSolution expected = {moves: 1, goalBucket: "two", otherBucket: 0};
+    test:assertEquals(result, expected);
+}
+
+@test:Config {
+    enable: false
+}
+function test6() {
+    // Measure using bucket one of size 2 and bucket two of size 3 - start with bucket one and end with bucket two
+    TwoBucketResult result = measure(
+        bucketOne = 2,
+        bucketTwo = 3,
+        goal = 3,
+        startBucket = "one"
+    );
+    TwoBucketSolution expected = {moves: 2, goalBucket: "two", otherBucket: 2};
+    test:assertEquals(result, expected);
+}
+
+@test:Config {
+    enable: false
+}
+function test7() {
```

Comment:
Maybe the test names could be a bit more descriptive? Especially for the ones that expect an error.